src/app/components/upload-dialog/file-uploading-dialog.component.scss

```diff
@@ -0,0 +1,94 @@
+@mixin adf-upload-dialog-theme($theme) {
```

Comment:
Do we need this scss in ACA?

Reply:
yes, we need to apply our colour over some elements

Reply:
do we need all that CSS to be in the ACA?

Reply:
apologies, was seeing other file. this one will be removed